components/Chat/Chat.tsx

```diff
@@ -200,15 +200,18 @@ export const Chat = memo(({ stopConversationRef }: Props) => {
             }
           }
           saveConversation(updatedConversation);
+
+          let conversationExists = false;
```

Comment:
Allows for adding a new conversation once a first message comes back from OpenAI. Hooking into the map allows for it to happen without duplication.